<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Symphonium</class>
 <widget class="QMainWindow" name="Symphonium">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Symphonium</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QSplitter" name="horMainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableWidget" name="libraryTableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
      <widget class="DisplayArea" name="displayArea" native="true"/>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <widget class="QSlider" name="slider">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_File_To_Lib"/>
    <addaction name="actionAdd_Folder_To_Lib"/>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>Play</string>
    </property>
    <addaction name="actionPlay_Pause"/>
    <addaction name="actionStop"/>
    <addaction name="actionSkip_Fwd"/>
    <addaction name="actionSkip_Back"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLibrary_Panel"/>
    <addaction name="separator"/>
    <addaction name="actionFull_Screen"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionselect_MIDI_device"/>
    <addaction name="separator"/>
    <addaction name="actionColors"/>
    <addaction name="actionPreferences_incoming"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOnline_FAQ"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlay"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAdd_File_To_Lib">
   <property name="text">
    <string>Add File To Lib</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionAdd_Folder_To_Lib">
   <property name="text">
    <string>Add Folder To Lib</string>
   </property>
  </action>
  <action name="actionPlay_Pause">
   <property name="text">
    <string>Play/Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionSkip_Fwd">
   <property name="text">
    <string>Skip Fwd</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionSkip_Back">
   <property name="text">
    <string>Skip Back</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionMain_Toolbar">
   <property name="text">
    <string>Main Toolbar</string>
   </property>
  </action>
  <action name="actionLibrary_Panel">
   <property name="text">
    <string>Library Panel On/Off</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full Screen On/Off</string>
   </property>
  </action>
  <action name="actionPreferences_incoming">
   <property name="text">
    <string>Preferences (incoming ..)</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionOnline_FAQ">
   <property name="text">
    <string>Online FAQ</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionselect_MIDI_device">
   <property name="text">
    <string>select MIDI device</string>
   </property>
  </action>
  <action name="actionColors">
   <property name="text">
    <string>Colors</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayArea</class>
   <extends>QWidget</extends>
   <header>displayarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
